mmap addr (#1870)
nix-rust/nix: CHANGELOG.md

- The MSRV is now 1.56.1
  ([#1792](https://github.com/nix-rust/nix/pull/1792))

> - Passes `addr` to `sys::mmap` as `Option<NonZeroUsize>` from `*mut c_void`.

Reviewer: I find this wording confusing.  How about this?
```suggestion
- The `addr` argument to `sys::mman::mmap` is now of type `Option<NonZeroUsize>`.
```

Author: Updated.
